dashboard/test/ui/test_status.js

```diff
@@ -72,14 +72,16 @@ function Test(fromRow) {
 Test.prototype.setLastModified = function (lastModified) {
```

Comment:
`Test.setLastModified` must be updated to return a `Promise` so that we can wait on the update if needed.